db_lib.py

```diff
@@ -663,13 +663,29 @@ def popNextRequest(beamlineName):
     actually pop it off the stack
     """
     orderedRequests = getOrderedRequestList(beamlineName)
+    logger.info(f"Requests in queue: {len(orderedRequests)}")
+    request_proposal_ids = set()
+    request_paths = set()
+    for req in orderedRequests:
+        if req["priority"] > 0 and req["priority"] != 99999:
+            request_proposal_ids.add(req["proposalID"])
+            request_paths.add(req["request_obj"]["basePath"])
+    visit_dir_path = Path(getBeamlineConfigParam(beamlineName, "visitDirectory")).resolve()
+    if request_proposal_ids:
+        if len(request_proposal_ids) > 1 and "commissioning" not in visit_dir_path.parts:
+            # Multiple proposal requests being run and not in commissioning
+            logger.info(f"Visit dir parents: {visit_dir_path.parts}")
+            return None
```

Comment:
the default return type appears to be {}, as seen at the end of the function. any reason why this is different? (also applies for the following elif)